src/conn_interface.c

```diff
@@ -231,6 +231,9 @@ void createSocketConnection(void (* initKeypress)())
     nopoll_ctx_unref(ctx);
     nopoll_cleanup_library();
     curl_global_cleanup();
+    clear_metadata();
+    rdk_logger_deinit();
```

Comment:
Facing build error due to this in local Ubuntu pc,

/usr/bin/ld: CMakeFiles/parodus.dir/conn_interface.c.o: in function `createSocketConnection':
conn_interface.c:(.text+0x89b): undefined reference to `rdk_logger_deinit'
collect2: error: ld returned 1 exit status
make[2]: *** [src/CMakeFiles/parodus.dir/build.make:519: src/parodus] Error 1
make[1]: *** [CMakeFiles/Makefile2:1274: src/CMakeFiles/parodus.dir/all] Error 2
make: *** [Makefile:141: all] Error 2

Reply:
@shilpa24balaji Could you please help to check this ? 
rdk_logger_deinit func is not available in local pc code hence this error.
For now we are proceeding by disabling this func call from conn_interface.c